### PR TITLE
Add Screenshots to Code Repo Token

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: docs/book/starter-guide/production-fundamentals/code-repositories.md [hunk 1]
```diff
@@ -103,10 +103,16 @@ each pipeline run.
 
 <summary>How To get a token for GitHub</summary>
 
-1. Go to your GitHub account settings and click on "Developer settings".
+1. Go to your GitHub account settings and click on "Developer settings". Or click [here](https://github.com/settings/tokens?type=beta)
```
Comment (strickvl) [inline docs/book/starter-guide/production-fundamentals/code-repositories.md:106]:
```suggestion
1. Go to your GitHub account settings and click on ["Developer settings"](https://github.com/settings/tokens?type=beta).
```

File: docs/book/starter-guide/production-fundamentals/code-repositories.md [hunk 2]
```diff
@@ -103,10 +103,16 @@ each pipeline run.
...
 2. Select "Personal access tokens" and click on "Generate new token".
-3. Give your token a name and select the scopes that you need (e.g. `repo`, `admin:repo_hook`, `user`).
+![Generate Fine-Grained Token](../../assets/code_repo/github-generate-fine-grained-token.png)
```
Comment (strickvl) [inline docs/book/starter-guide/production-fundamentals/code-repositories.md:108]:
```suggestion
![Generate a fine-grained token](../../assets/code_repo/github-generate-fine-grained-token.png)
```

File: docs/book/starter-guide/production-fundamentals/code-repositories.md [hunk 3]
```diff
@@ -103,10 +103,16 @@ each pipeline run.
...
+3. Give your token a name and description.
+![Give Name and Description](../../assets/code_repo/github-fine-grained-token-name.png)
+4. We recommend select specific repository and then give `contents`read only access.
+![Set Permessions](../../assets/code_repo/github-fine-grained-permessions.png)
```
Comment (strickvl) [inline docs/book/starter-guide/production-fundamentals/code-repositories.md:112]:
```suggestion
![Set Permissions](../../assets/code_repo/github-fine-grained-permissions.png)
```
(and fix the filename)

File: docs/book/starter-guide/production-fundamentals/code-repositories.md [hunk 4]
```diff
@@ -139,9 +145,11 @@ each pipeline run.
 
 <summary>How To get a token for GitLab</summary>
 
-1. Go to your GitLab account settings and click on "Access Tokens".
-2. Select the scopes that you need (e.g. `read_repository`, `write_repository`).
+1. Go to your GitLab account settings and click on "Access Tokens". Or click [here](https://gitlab.com/-/profile/personal_access_tokens).
+2. Give a Name to the token and select the scopes that you need (e.g. `read_repository`, `read_user`, `read_api` )
+![GitLab Token generate](../../assets/code_repo/gitlab-generate-access-token.png)
```
Comment (strickvl) [inline docs/book/starter-guide/production-fundamentals/code-repositories.md:150]:
```suggestion
![Generate the Gitlab token](../../assets/code_repo/gitlab-generate-access-token.png)
```

File: docs/book/assets/code_repo/github-fine-grained-permessions.png
Comment (strickvl) [inline docs/book/assets/code_repo/github-fine-grained-permessions.png:1]: this filename also needs updating

**Review by strickvl — CHANGES_REQUESTED** [review]

**Review by schustmi — APPROVED** [review]

**Pull request merged**